### PR TITLE
Worst loss

### PR DESCRIPTION
**Pull request by Paul-Knapp**
worst loss method works and handles nils. tests have been written to verify all functionality.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -340,4 +340,17 @@ def average_win_percentage(team_id)
     wins > 0 ? ((wins.to_f / games)).to_f.round(2) : 0
   end
 
+  def worst_loss(team_id) 
+    worst_loss_margin = []
+
+    @all_games.each do |game|
+      # binding.pry
```
Comment (Crosswolfv1) [inline lib/stat_tracker.rb:347]: remove this please, otherwise code looks good and functions well

**Review by Crosswolfv1 — CHANGES_REQUESTED** [review]
Other than the binding.pry the code looks good and is functional, good tests too

**Review by wally-yawn — APPROVED** [review]

**Review by Montana-Pfeifer — COMMENTED** [review]
methods look good and get proper losses

File: lib/stat_tracker.rb (hunk 1 above)
Comment (Montana-Pfeifer) [reply lib/stat_tracker.rb:347]: it has been removed

**Pull request merged**